make.config

```diff
@@ -23,8 +23,9 @@ WITH_SOCKETS := 1
 WITH_ARPACK := 1
 
 # Whether the DFTD3 library (dispersion) should be linked.
-# NOTE: Due to license incompatibility, distribution of a DFTB+ binary built
-# with this component is not permitted. Use it only for your personal research.
+# NOTE: Due to the license of the DFTD3 library, the linked code must be
```

Comment:
Suggest 'combined code' instead of 'linked code'

Reply:
Done.